examples/6tisch/simple-node/Makefile

```diff
@@ -19,6 +23,23 @@ MODULES += $(CONTIKI_NG_SERVICES_DIR)/shell
 
 ifeq ($(MAKE_WITH_ORCHESTRA),1)
 MODULES += $(CONTIKI_NG_SERVICES_DIR)/orchestra
+
+ifeq ($(MAKE_WITH_LINK_BASED_ORCHESTRA),1)
+ifeq ($(MAKE_WITH_STORING_ROUTING),1)
+CFLAGS += -DORCHESTRA_CONF_RULES="{&eb_per_time_source,&unicast_per_neighbor_link_based,&default_common}"
+else
+$(error "Inconsistent configuration")
+endif # MAKE_WITH_LINK_BASED_ORCHESTRA
+endif # MAKE_WITH_STORING_ROUTING
+
+endif # MAKE_WITH_ORCHESTRA
+
+ifeq ($(MAKE_WITH_STORING_ROUTING),1)
+MAKE_ROUTING = MAKE_ROUTING_RPL_CLASSIC
+CFLAGS += -DRPL_CONF_WITH_STORING=1
+ifneq ($(MAKE_WITH_LINK_BASED_ORCHESTRA),1)
+CFLAGS += -DORCHESTRA_CONF_RULES="{&eb_per_time_source,&unicast_per_neighbor_rpl_storing,&default_common}"
+endif
```

Comment:
Can't the conditions above be re-factored for readability? The first checks link-based then storing, the second is outside of block `ifeq ($(MAKE_WITH_ORCHESTRA),1)` yet it configures Orchestra, and checks storing then link-based. Looks like as bug even, or am I missing something?

Reply:
Refactored